pkg/model/crd.go

```diff
@@ -387,6 +387,34 @@ func (r *CRD) GetOutputWrapperFieldPath(
 	return &opConfig.OutputWrapperFieldPath
 }
 
+// OutputShapeName returns name of the Output shape for given operation.
+func (r *CRD) UnwrappedOutputShapeName(
+	// The operation to look for the Output shape
+	op *awssdkmodel.Operation,
+) *string {
```

Comment:
Thinking more about this, i believe we should return the `outputShape` instead of the `outputShape.Name`. This way we can reuse this function other places as well like: https://github.com/aws-controllers-k8s/code-generator/blob/main/pkg/generate/code/set_resource.go#L138-L145

Reply:
Changed the return type. However, did not update the set_resource.go as that also sets `sourceVarName` var thus would need to keep its logic.